### PR TITLE
update docs to reflect separate mirror and rc tiles

### PR DESCRIPTION
**Pull request by drlau**
Note: There's 2 TODO lines marked with my name. I will revisit these in a separate PR once I confirm these and update the tile field names. If you prefer that the TODO's are addressed in this PR, let me know.

### CODE REVIEW

File: install.html.md.erb [hunk 1]
```diff
@@ -19,19 +19,21 @@ For compatible versions, see the [Product Snapshot](./index.html#snapshot).
...
+
+If you do not have an external mirror for ClamAV jobs to fetch databases updates from, you can deploy one using the ClamAV Mirror for PCF.
+This deployed mirror can support air gapped environments.
 
 1. Download the product file from [Pivotal Network](https://network.pivotal.io/products/p-clamav-addon).
 
 1. Navigate to the Ops Manager Installation Dashboard and select **Import a Product** to upload the product file.
 
-1. Under the **Import a Product** button, click **+** next to the version number of <%= vars.product_short %>.
+1. Under the **Import a Product** button, click **+** next to the version number of ClamAV Mirror for PCF.
```
Comment (simonxca) [inline install.html.md.erb]: How come we're not using the `erb` templating anymore?
I see `<%= vars.product_full %>` above.

File: install.html.md.erb [hunk 2]
```diff
@@ -56,12 +58,109 @@ To assign AZs and networks, click the **Assign AZs and Networks** settings tab.
...
+to one of the following supported options:
+
+* `stdout`: sends messages to /var/vcap/sys/log/clamav-mirror/clamav-mirror.stdout.log.
+* `stderr`: sends messages to /var/vcap/sys/log/clamav-mirror/clamav-mirror.stderr.log.
+* `syslog`: sends messages to /var/log/messages.
+
+1. The ClamAV Mirror for PCF will serve virus definitions to your environment for the ClamAV Add-on for PCF to use, but the ClamAV Mirror needs to get databases itself.
+You can configure the ClamAV Mirror to get virus definitions from one of the following supported options:
+
+<%# TODO Dylan: less verbose things with tile field updates %>
```
Comment (simonxca) [inline install.html.md.erb]: Still in progress?

File: install.html.md.erb [hunk 3]
```diff
@@ -56,12 +58,109 @@ To assign AZs and networks, click the **Assign AZs and Networks** settings tab.
...
+Then enter the `host`, `port`, `username` and `password` in the fields that appear.
+
+###<a id="syslog"></a> Configure System Logging
+Follow the steps below to enable system logging for ClamAV Mirror.
+Logs use RFC5424 format.
+
+1. Click **Syslog**.
+1. Click **Yes**.
+
+    <<<SCREENSHOT>>>
```
Comment (simonxca) [inline install.html.md.erb]: Screenshot will be supplied by docs team?

File: install.html.md.erb (hunk 1 above)
Comment (Pivotal-Christopher-Wong) [reply install.html.md.erb]: <%= vars.product_full %> to be used on first mention on a page, <%= vars.product_short %> to be used on subsequent mentions on that same page.

File: install.html.md.erb (hunk 3 above)
Comment (Pivotal-Christopher-Wong) [reply install.html.md.erb]: Will coordinate closer to release when the tile UI fields are finalized to grab any necessary screenshots required.

File: install.html.md.erb (hunk 2 above)
Comment (Pivotal-Christopher-Wong) [reply install.html.md.erb]: Still in progress. Was submitted to us near end of day, and I was OOO this morning. Hope to have this at least merged by EOD.

**Comment by Pivotal-Christopher-Wong** [comment]
Thanks for the PR @drlau !

**Pull request merged**